src/main/resources/assets/admin/common/js/ui/time/Picker.ts

```diff
@@ -47,6 +55,13 @@ export class Picker<T extends Element>
         this.input.resetBaseValues();
     }
 
+    public setDefaultHandler(setDefault: Function, defaultDate: Date | undefined): void {
```

Comment:
Rename method to `setDefaultValueHandler`, rename `setDefault` argument to `handler`, remove `defaultDate`